plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb.test.ts

```diff
@@ -244,6 +249,85 @@ describe('MongoDBInstrumentation', () => {
     });
   });
 
+  describe('when specifying a responseHook configuration', () => {
+    const dataAttributeName = 'mongodb_data';
+    beforeEach(() => {
```

Comment:
Are you sure this is necessary? Isn't that taken care of by the top-level describe `beforeEach`?

Reply:
It is done in every 2nd level describe - same level we are adding here.
Not going to refactor their tests now ;)